var/spack/repos/builtin/packages/py-atlannot/package.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.build_systems.python import PythonPackage
+from spack.directives import depends_on, version
+
+
+class PyAtlannot(PythonPackage):
+    """Alignment of brain atlas annotations."""
+
+    homepage = 'https://bbpgitlab.epfl.ch/project/proj101/atlas_annotation'
+    git = 'git@bbpgitlab.epfl.ch:project/proj101/atlas_annotation.git'
+
+    maintainers = ['EmilieDel', 'Stannislav']
+
+    version('0.1.1', commit='5271681de15f23e1df6f8556405b03218541d241')
+
+    depends_on('python@3.7:', type=('build', 'run'))
+    depends_on('py-setuptools', type='build')
+    depends_on('py-setuptools-scm', type='build')
+
+    # Installation requirements
+    depends_on('py-antspyx@0.2.4', type=('run'))
+    depends_on('py-atldld@0.2.2', type=('run'))
+    # depends_on('py-dvc', type=('run'))
```

Comment:
What's up with this?

Reply:
Ah yes, it's a package that's in theory listed in the requirements in `setup.py`, but it's not really needed to use the library, only to manually fetch the input data.

So, I could do one of these things:
- Delete this line
- Deploy `py-dvc`, but like I said, we don't really need this now
- Remove this dependency from the install requirements of the library and maybe put it inside `extra_requires`.

Reply:
If it's in `setup.py`, then our distribution check should have complained. A comment may be nice, but for now I'm fine with it as it is.

Reply:
Yes, if it's OK, let's leave it there as a reminder. I'll also create a ticket on our JIRA to do something about this requirement in the future.